Task2/domain/src/main/java/com/epam/esm/repository/type/SortTypeQuery.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.esm.repository.type;
+
+public enum SortTypeQuery {
```

Comment:
Same, why not private static final String?

Reply:
I ended up using enum for such situations